src/views/RecordPage.vue

```diff
@@ -20,15 +20,19 @@
       <ion-item>
         <WeekHeader :highlighted-day-index="5" />
       </ion-item>
+      <div v-if="!routines.length">
+        Click bottom right + to add new routine.
+      </div>
       <ion-list>
         <WeekItem
-          v-for="header in ['Foo', 'Bar', 'Woo']"
+          v-if="routines.length"
+          v-for="header in routines.map((r) => r.name)"
```

Comment:
Why using `map()` here?

Reply:
`header` is an array of string so need to map routines.name into the same type